src/util/transition/util.ts

```diff
@@ -0,0 +1,7 @@
+export function getTransitionDurationMs(element: HTMLElement) {
+  const {transitionDelay, transitionDuration} = window.getComputedStyle(element);
+  const transitionDelaySec = parseFloat(transitionDelay);
+  const transitionDurationSec = parseFloat(transitionDuration);
+
```

Comment:
We need to be careful with the returned value. At the moment, there is only one duration and delay with the Bootstrap transitions, but the value returned can be multiple, for example : "1s, 0.25s" (in case of of a custom theme for example).

Reply:
For this one I just copied the bootstrap's behaviour, they pick the first one for some reason. I haven't investigated further, but added a unit test for this: https://github.com/ng-bootstrap/ng-bootstrap/pull/3716/files#diff-249b446faed753f89f2e46d4f08fef51R44-R47